src/content/api/_endpoints/promotions-list-by-loyaltyProgramId.js

```diff
@@ -0,0 +1,48 @@
+export default {
+  method: 'GET',
+  path: '/api/loyalty-programs/WRhAxxWpTKb5U7pXyxQjjY/promotions',
+  request: {
+    headers: {
+      'X-Api-Key': '<TOKEN>',
+    },
+  },
+  response: {
+    items: [
+      {
+        id: '8aoMfscvtuewsuJzmzBzAs',
+        accountId: '57SyRRgZ1U8Kq2wKpCnSR5',
+        programId: 'WRhAxxWpTKb5U7pXyxQjjY',
+        name: 'Spend a Buck',
+        summary: 'Make a payment',
+        description: 'The amount of the payment must be greater than $1.',
+        mediaUploadId: '26yUWG6wFgmva8UaDiCTWq',
+        img: 'https://media-upload.centrapay.com/image.png?jhbdsfau67ewejshb=487hsdjhbdgs743',
+        startsAt: '2023-02-16T00:47:54.131Z',
+        endsAt: '2024-02-16T00:47:54.131Z',
+        rewards: [
+          {
+            assetType: 'centrapay.nzd.main',
+            amount: '500',
+          }
+        ],
+        eventType: 'payment',
+        target: {
+          type: 'count',
+          amount: 1
+        },
+        type: 'challenge',
+        conditions: [
+          {
+            field: 'amount',
+            value: '99',
```

Comment:
If the description is "The amount of the payment must be greater than $1." then I believe this value should be 100.

Reply:
99 is correct. `greater-than` operator is not inclusive

Reply:
Sorry, Laurence is correct. Since the payment amount must be greater than $1 (100 cents), we should put `100`